pkg/api/controller.go

```diff
@@ -817,12 +841,27 @@ func (c *Controller) CreateUser(w http.ResponseWriter, r *http.Request, body Cre
 		FriendlyName: nil,
 		Source:       "internal",
 	}
+	normalizedEmail := normalizeAndSetToLowercase(body.Id)
```

Comment:
1. At this point this is also a user-provided data - don't normalize it here
2. You need to extract the email - verify it is an email and lower case (no need for the utf8 norm) before you check permission or initialize the user.
3. The logic in this pr changes how we treat user email at the email field - until now we respected the email case and kept it inside the user's email.

Reply:
Got it. fixed